README.md

```diff
@@ -866,6 +919,17 @@ The custom options are:
   When disabled, it will forward the user to the cross-device flow in order to attempt to capture a video in another device. If the user is already in a mobile device and it does not support
   MediaRecorder, the unsupported browser error will be shown.
 
+#### auth
+
+This is Authentication step. If you have followed the guidelines specific to including auth, you'll have this step made available. In here, a loading screen is presented to the user to fetch all necessary resources to perform authentication.
+
+After all resources are loaded, the session is initialised, and the authentication check begins. An oval frame of the camera will be present (if camera permissions are provided) and actionable elements will render, asking the user to place their face in frame, followed up by a different set of instructions for them to follow to successfully authenticate the user.
+
+If the user is not a match, or conditions are not good to successfuly authenticate, they will be asked to retry authentication. If authentication is not possible, the page will rollback to the previous step.
+
+**The Authentication module is currently a BETA feature.**
```

Comment:
I would place this higher up, and make a note with a ⚠️ emoji, consistently with other similar notices

Reply:
I placed here following on poa's beta feature warning. Should I put it right at the first place it's mentioned?

Reply:
Yes I would do

Reply:
Added it at the `4.1 HTML Script Tag Include` where auth is mentioned for the first time in the README